src/languages/yaml.js

```diff
@@ -42,14 +42,28 @@ export default function(hljs) {
       }
     ]
   };
-  const STRING = {
+
+  const SINGLE_QUOTE_STRING = {
     className: 'string',
     relevance: 0,
     variants: [
       {
         begin: /'/,
         end: /'/
-      },
+      }
+    ],
+    contains: [
+      {
+        begin: /''/,
```

Comment:
I wonder if this should be `char.escape` scoped?

Reply:
if we use `char.escape` scope both quotes are included
```
<span class="hljs-char escape_">&#x27;&#x27;</span>
```
should I change the pattern to target only the first quote in the pair?

Reply:
> both quotes are included

Makes sense to me, I'd say both quotes are the entire escape sequence - which is what we're wanting to make stand out...